askama_derive/src/lib.rs

```diff
@@ -11,14 +11,29 @@ use proc_macro2::Span;
 mod config;
 mod generator;
 mod heritage;
+#[cfg(feature = "localization")]
+mod i18n;
 mod input;
 mod parser;
 
-#[proc_macro_derive(Template, attributes(template))]
+#[proc_macro_derive(Template, attributes(template, locale))]
 pub fn derive_template(input: TokenStream) -> TokenStream {
     generator::derive_template(input)
 }
 
+#[proc_macro]
```

Comment:
Why do we need another macro for this?

Reply:
This indirection helps the user with a friendly compiler error if the necessary feature is not enabled.

Should this macro be removed, and the `i18n::derive` macro be called directly?